sdk/ml/azure-ai-ml/azure/ai/ml/constants/_common.py

```diff
@@ -67,6 +67,8 @@
     "Only AmlCompute/KubernetesCompute cluster properties are supported, compute name {}, is {} type."
 )
 MAX_AUTOINCREMENT_ATTEMPTS = 3
+MAX_ASSET_STORE_API_CALL_ATTEMPTS = 3
+REGISTRY_DISCOVERY_BASE_URI = "https://eastus.api.azureml.ms"
```

Comment:
Consider using this : https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ml/azure-ai-ml/azure/ai/ml/_azure_environments.py#:~:text=EndpointURLS.REGISTRY_DISCOVERY_ENDPOINT%3A%20%22https%3A//eastus.api.azureml.ms/%22%2C

Reply:
Thanks so much!